bin/pipeline_result_processor/prp/models/phenotype.py

```diff
@@ -0,0 +1,76 @@
+"""Datamodels used for prediction results."""
+from enum import Enum
+from typing import Dict, List
+
+from pydantic import BaseModel, Field
+
+from .base import RWModel
+
+
+class VariantType(Enum):
+    substitution = "substitution"
```

Comment:
Consider re-work how constants are handled, for example consider having a separate file named "constants.py" where constants are set, then imported (so everything can be managed from one place and code will be easier to maintain). Also consider using a naming convention with capital letters like `SUBSTITUTION = "substitution"`

Reply:
I agree with capitalized constants but I do think its easier to maintain when you group all models based on the type of data they models.

For instance are the class VariantType only used when describe the data format of a Variant. I think it makes more sense of keeping these to gather rather than having every constant in the script in a "constants" file regardless of where and how they werre used.